src/platform/forms/components/common/alerts/ErrorMessage.jsx

```diff
@@ -0,0 +1,33 @@
+// libs
+import React from 'react';
+
+/**
+ * A column layout component
+ * @param {boolean} active when true, renders the component
+ * @param {node} children content that the columns class wraps
+ * @param {string} message error message text to appear
+ * @param {string} title error title text to appear
+ */
+function ErrorMessage(props) {
+  const { active, children, message, testId, title } = props;
+
+  if (!active) return null;
+  else {
+    return (
+      <div
+        className="usa-alert usa-alert-error schemaform-failure-alert"
+        data-testid={testId}
+      >

```

Comment:
The answer may be no, but does it make sense to use [Formation's `AlertBox`](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/alertbox/) either here in `ErrorMessage` or in place of it?

Reply:
I think that is the end goal. We have competing styles in the forms library. By starting to extract this as a reusable component, we give ourselves the opportunity to align to a source of truth and make it easier to update to that pattern.